client/blocks/edit-gravatar/style.scss

```diff
@@ -48,3 +56,30 @@
 .edit-gravatar-modal .dialog__content {
 	height: 100%;
 }
+
+.edit-gravatar__label-container {
+	position: absolute;
+		left: 0;
+		top: 0;
```

Comment:
nice indentation - I often forget to do this even though it's in our style guide